<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormInfoWindows</class>
 <widget class="QWidget" name="FormInfoWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>307</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Windows</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblChilds">
     <property name="text">
      <string>Child windows:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSiblings">
     <property name="text">
      <string>Sibling windows:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblParent">
     <property name="text">
      <string>Parent window:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="txtParentClassname">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QLineEdit" name="txtParentWindowtext"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblOwner">
     <property name="text">
      <string>Owner window:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="txtOwnerClassname">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QLineEdit" name="txtOwnerWindowtext"/>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTreeView" name="treeviewChilds">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QTreeView" name="treeviewSiblings">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QVBoxLayout" name="layout_link_parent"/>
   </item>
   <item row="5" column="1">
    <layout class="QVBoxLayout" name="layout_link_owner"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>txtOwnerWindowtext</sender>
   <signal>returnPressed()</signal>
   <receiver>FormInfoWindows</receiver>
   <slot>slotLineEditReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtParentWindowtext</sender>
   <signal>returnPressed()</signal>
   <receiver>FormInfoWindows</receiver>
   <slot>slotLineEditReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeviewChilds</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>FormInfoWindows</receiver>
   <slot>slotTreeViewDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeviewSiblings</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>FormInfoWindows</receiver>
   <slot>slotTreeViewDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotLineEditReturnPressed()</slot>
  <slot>slotTreeViewDoubleClicked(QModelIndex)</slot>
 </slots>
</ui>
